packages/loot-core/src/server/api.ts

```diff
@@ -614,6 +614,17 @@ handlers['api/category-group-update'] = withMutation(async function ({
   });
 });
 
+handlers['api/apply-multiple-templates'] = withMutation(async function ({
```

Comment:
I don't think we need to change the api here.  The other template actions use loot-core/src/server/budget/app.ts for the action.